Gemfile.lock

```diff
@@ -373,4 +373,4 @@ RUBY VERSION
    ruby 2.3.7p456
 
 BUNDLED WITH
-   1.16.6
+   1.17.1
```

Comment:
This PR is for upgrading `loofah` and it's upgrading the bundle version in the lock file. It's fine with me I just want to check if you notice this.

Reply:
yeah, my bundler version was < bundler version in the lockfile, but when I upgraded it became > bundler version in the lockfile so I updated it.

I don't think it should have any consequences, but I can remove if you think that it might